plugins/BEdita/API/tests/TestCase/Controller/LoginControllerTest.php

```diff
@@ -76,19 +87,26 @@ public function testSuccessfulRenew(array $meta)
     }
 
     /**
-     * Test login method with invalid credentials.
+     * Test login method with invalid credentials or wrong content type
      *
      * @return void
      *
      * @covers ::login()
      */
     public function testFailedLogin()
     {
-        $this->configRequestHeaders('POST');
+        $this->configRequestHeaders('POST', ['Content-Type' => 'application/json']);
 
         $this->post('/auth', ['username' => 'first user', 'password' => 'wrongPassword']);
 
         $this->assertResponseCode(401);
+
+        // using default 'application/vnd.api+json'
```

Comment:
I'd prefer to split these tests in two, one with invalid credentials, the other with wrong content type. No data providers: just two test cases for better understanding of the problem in case it fails. Same for `testSuccessfulLogin` test case, first with a content type, then with the other one.

However, this is a very minor concern.